ext_emconf.php

```diff
@@ -4,8 +4,8 @@
     'title' => 'Authenticator',
     'description' => 'Implements a two factor authentication for TYPO3. Currently a for backend only.',
     'category' => 'services',
-    'author' => 'Philipp Gampe',
-    'author_email' => 'philipp.gampe@typo3.org',
+    'author' => 'Philipp Gampe, Oliver Eglseder',
```

Comment:
Those fields are single value only. You are a very valuable contributor, but not the author ;)